pages.es/linux/xsel.md

```diff
@@ -0,0 +1,28 @@
+# xsel
+
+> Herramienta X11 de selección y manipulación del portapapeles.
+> Más información: <https://manned.org/xsel>.
+
+- Utiliza la salida de un comando como entrada del portapapeles (clip[b]oard) (equivalente a `Ctrl + C`):
+
+`echo 123 | xsel -ib`
+
+- Utiliza el contenido de un archivo como entrada del portapapeles:
+
+`cat {{ruta/al/archivo}} | xsel -ib`
+
+- Produce el contenido del portapapeles en el terminal (equivalente a `Ctrl + V`):
+
+`xsel -ob`
+
+- Envía el contenido del portapapeles a un archivo:
+
+`xsel -ob > {{ruta/al/archivo}}`
+
+- Limpia el portapapeles:
+
+`xsel -cb`
+
+- Envía el contenido de la selección primaria X11 a la terminal (equivalente a un clic medio del ratón):
```

Comment:
confusing the term `clic medio del ratón`

Reply:
Sure, I updated to put the third button instead of middle one, hope it clarifies.